packages/pwncat/PKGBUILD

```diff
@@ -24,4 +24,3 @@ package() {
 
   cp --no-preserve=ownership -a pse "$pkgdir/usr/share/doc/$pkgname/"
 }
-
```

Comment:
Keep the final newline.

Reply:
Fixed. Just out if curiousity, why is this required?

Reply:
It's because packaging style, nothing required to build/work but we maintain this.